Website/htdocs/mpmanager/app/Sms/Bongo.php

```diff
@@ -77,15 +77,16 @@ public function sendGetSms(string $number, string $body)
     }
 
     /**
+     *
      * @param string $number
      * @param string $body
      * @param string $callback
-     *
+     * @param SmsAndroidSetting $smsAndroidSetting
      * @return mixed|StreamInterface
      *
      * @throws \Exception
      */
-    public function sendSms(string $number, string $body, $callback)
+    public function sendSms(string $number, string $body, $callback, \App\Models\SmsAndroidSetting $smsAndroidSetting)
```

Comment:
Could you import `SmsAndroidSetting` here instead of giving the absolute path?

Reply:
sure thing